### DISCUSSION TITLE
BakeMaster v3.0 is in progress 🔩⚙️

### DISCUSSION THREAD

**Discussion by KirilStrezikozin** [KirilStrezikozin/BakeMaster-Blender-Addon#5] [Announcements]
Hi, everyone 👋
I'm announcing **BakeMaster 2.0 update** that is currently in development 🔩⚙️

This update will bring up the strong sides of BakeMaster addon and make it much better from all aspects!
List of what is going to be delivered:

- [x] Presets for settings panels - 🎯
- [x] New map: ColorID - 🎯
- [x] New map: Vector Displacement - 🎯
- [x] Texture Sets
- [x] Channel Packing
- [x] Smoothing Groups
- [x] Decals Baking
- [x] And documentation reworks in addition - 🎯

The current stage is getting UI done.

(When you see all these categories checked, it'll mean that BakeMaster 2.0 is about to be released in the soonest time)

To propose your own ideas on what to add to BakeMaster, be free to write to [💡Ideas](https://github.com/KirilStrezikozin/BakeMaster-Blender-Addon/discussions/new?category=ideas) or [Submit issue](https://github.com/KirilStrezikozin/BakeMaster-Blender-Addon/issues/new/choose), or send a message to [💡 Ideas in our BakeMaster Discord Server](https://discord.gg/2ePzzzMBf4)!

**Comment by sangouzhi** [comment]
Hopefully the ao baked results will be as good as the Marmoset Toolbag 4, although this is a bit tricky

**Reply by KirilStrezikozin** [reply]
hi, check out #21, maybe you've got something to add)

**Comment by KirilStrezikozin** [comment]
**Hi, bakers 👋**
BakeMaster 2.0 brought a lot of cool things, but it wasn't the end!
I'm here to announce that **BakeMaster 3.0 is now under dev**
and brings new professional features to the addon! Have a look at them all [here](https://github.com/KirilStrezikozin/BakeMaster-Blender-Addon/issues).
